src/cmd-build

```diff
@@ -252,6 +266,11 @@ for itype in "${IMAGE_TYPES[@]}"; do
               # TODO: Remove this, things should be parsing the metadata
               ln -s "${img_qemu}" "${name}"-qemu.qcow2
               ;;
+        metal-bios)
+            img_metalbios=${imageprefix}-metal-bios.raw
+            images[$itype]="${img_metalbios}"
+            run_virtinstall "$(pwd)"/"${img_metalbios}" --variant=metal
```

Comment:
do we need to do anything with setting an oemid here? I guess we haven't decided that yet pending: https://github.com/coreos/fedora-coreos-tracker/issues/142

Reply:
Yeah; I vote Benjamin picks it.

Reply:
Oh he's on PTO.  This doesn't need to be a blocker though I think.

Reply:
+1

Reply:
`metal` SGTM :grin: